### DISCUSSION TITLE
Lynx Distributor fuse monitor via I2C

### DISCUSSION THREAD

**Discussion by BjoernKellermann** [shining-man/bsc_fw#160] [Ideas]
Ist would be great of we could use the I2C interface to integrate with a Victron Lynx Distributor to get information about fuse status.

Thomas implemented a dbus driver for an sum to I2C converter which might help as starting point:
https://github.com/twam/dbus-lynx-distributor

**Comment by shining-man** [comment]
Theoretically, it should be possible. The problem is the voltage. The BSC works with 3v3 and the Lync with 5v. You could use a level converter like this: https://www.makershop.de/module/schnittstellen/4-kanal-konverter/
With the addresses you would have to make sure that there are no problems with the display and the serial extension.

What should happen to the values when they are in the BSC?
